### PR TITLE
Updated static content

### PR DESCRIPTION
**Pull request by longhotsummer**
Pulling in changes from Deena.

@jbothma

### CODE REVIEW

File: umibukela/templates/about.html [hunk 1]
```diff
@@ -260,9 +260,9 @@ <h2 class="text-white">How do we do CBM?</h2>
           <nav class="steps-sidebar">
             <h4>The CBM Process</h4>
             <ul class="nav nav-stacked">
-              <li><a href="#government">1. Establish gov. relationship</a></li>
+              <li><a href="#government">1. Build Gov. relationship</a></li>
```
Comment (jbothma) [inline umibukela/templates/about.html]: should Gov be capitalised?

**Comment by jbothma** [comment]
Looks cool to me apart from one pedantic question

File: umibukela/templates/about.html (hunk 1 above)
Comment (longhotsummer) [inline umibukela/templates/about.html]: I think so, but perhaps I'm being inconsistent. As the name of an entity, Government is generally capitalised (as opposed to "the government"), so the shortening should be too. But we haven't done that everywhere.

Comment (longhotsummer) [inline umibukela/templates/about.html]: Although I could be wrong: http://www.chicagomanualofstyle.org/qanda/data/faq/topics/CapitalizationTitles/faq0015.html

**Comment by longhotsummer** [comment]
Ok, I changed that G, I'm going to merge.

**Pull request merged**